indico/modules/events/registration/models/registrations.py

```diff
@@ -342,7 +342,7 @@ def render_price(self):
         currency = event_payment_settings.get(self.registration_form.event, 'currency')
         return '{} {}'.format(self.price, currency)
 
-    def update_state(self, approved=None, paid=None, rejected=None):
+    def update_state(self, approved=None, paid=None, rejected=None, manager=False):
```

Comment:
I'd call it `management` (other functions in the module with similar management-sepcific behavior all use `management` for the argument)